plugin/trino-hive/src/main/java/io/trino/plugin/hive/procedure/SyncPartitionMetadataProcedure.java

```diff
@@ -191,7 +191,7 @@ private static Set<Location> listDirectories(TrinoFileSystem fileSystem, Locatio
     private static String listedDirectoryName(Location directory, Location location)
     {
         String prefix = directory.path();
```

Comment:
Update the description of your PR to point out that you're using the same strategy as in https://github.com/trinodb/trino/blob/f7369f674384b7cde8b5eb97931d679674930c2f/lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystem.java#L175-L177

Reply:
I haven't noticed the same strategy was already in use on another module..!

Thanks, I will update the description!

Reply:
squash the commits into one pls.